sonar-db/src/main/java/org/sonar/core/permission/ProjectPermissions.java

```diff
@@ -28,11 +29,11 @@
  * Holds the constants representing the various component permissions that can be assigned to users & groups
  *
  */
-public final class ComponentPermissions {
+public final class ProjectPermissions {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)